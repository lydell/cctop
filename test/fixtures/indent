#comment
 #comment
    # comment
   # comment
  # comment

  |
           |
	|

command      a        |
\b
\ b
\  b
\   b
\	b
 \c
       \   c         |
       #comment
       \d
#comment
  \d
  sub1
    sub2
      sub3
        sub4
    sub2
        sub3
    sub2
    sub2
