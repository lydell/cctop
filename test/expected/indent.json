{
  "commands": [
    {
      "name": "command",
      "value": "a        \nb\nb\n b\n  b\n\tb\nc\n  c         \nd\nd",
      "lines": [
        [11, 14, 9],
        [12, 2, 1],
        [13, 3, 1],
        [14, 3, 2],
        [15, 3, 3],
        [16, 2, 2],
        [17, 3, 1],
        [18, 10, 12],
        [20, 9, 1],
        [22, 4, 1]
      ],
      "line": 11,
      "column": 1,
      "endLine": 30,
      "subCommands": [
        {
          "name": "sub1",
          "value": "",
          "lines": [[23, 7, 0]],
          "line": 23,
          "column": 3,
          "endLine": 30,
          "subCommands": [
            {
              "name": "sub2",
              "value": "",
              "lines": [[24, 9, 0]],
              "line": 24,
              "column": 5,
              "endLine": 26,
              "subCommands": [
                {
                  "name": "sub3",
                  "value": "",
                  "lines": [[25, 11, 0]],
                  "line": 25,
                  "column": 7,
                  "endLine": 26,
                  "subCommands": [
                    {
                      "name": "sub4",
                      "value": "",
                      "lines": [[26, 13, 0]],
                      "line": 26,
                      "column": 9,
                      "endLine": 26,
                      "subCommands": []
                    }
                  ]
                }
              ]
            },
            {
              "name": "sub2",
              "value": "",
              "lines": [[27, 9, 0]],
              "line": 27,
              "column": 5,
              "endLine": 28,
              "subCommands": [
                {
                  "name": "sub3",
                  "value": "",
                  "lines": [[28, 13, 0]],
                  "line": 28,
                  "column": 9,
                  "endLine": 28,
                  "subCommands": []
                }
              ]
            },
            {
              "name": "sub2",
              "value": "",
              "lines": [[29, 9, 0]],
              "line": 29,
              "column": 5,
              "endLine": 29,
              "subCommands": []
            },
            {
              "name": "sub2",
              "value": "",
              "lines": [[30, 9, 0]],
              "line": 30,
              "column": 5,
              "endLine": 30,
              "subCommands": []
            }
          ]
        }
      ]
    }
  ],
  "errors": []
}